src/pip/_internal/wheel.py

```diff
@@ -500,16 +500,19 @@ def _get_script_text(entry):
         with open_for_csv(temp_record, 'w+') as record_out:
             reader = csv.reader(record_in)
             writer = csv.writer(record_out)
+            outrows = []
             for row in reader:
                 row[0] = installed.pop(row[0], row[0])
                 if row[0] in changed:
                     row[1], row[2] = rehash(row[0])
-                writer.writerow(row)
+                outrows.append(row)
```

Comment:
Make this `outrows.append(tuple(row))`. Otherwise py2 does some type-name based logic and py3 raises a TypeError:

```
TypeError: '<' not supported between instances of 'tuple' and 'list'
```

That should be enough to fix this. :)

Reply:
yay! That fixed the tests.